### PR TITLE
35 - Added embedded datastudio dashboard

### PR DESCRIPTION
**Pull request by mukulshakya**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

About # (link your issue here)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [x] I have added screenshots related to my pull request (for frontend tasks)
- [x] I have pasted a gif showing the feature.
- [x] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: src/pages/DatastudioDashboard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+`
+
+export default function DatastudioDashboard(): JSX.Element {
+    return (
+        <Wrapper>
+            <iframe
+                title="Datastudio Dashboard"
+                width="100%"
+                height="2000px"
+                src="https://datastudio.google.com/embed/reporting/f1ce8f56-058c-4e31-bfd4-1a741482642a/page/p_miuw33ljnc"
```
Comment (sirpy) [inline src/pages/DatastudioDashboard/index.tsx:24]: make this an env variable
have this as a default value in .env or some other idiomatic way

File: src/pages/DatastudioDashboard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    }
+`
+
+export default function DatastudioDashboard(): JSX.Element {
+    return (
+        <Wrapper>
+            <iframe
+                title="Datastudio Dashboard"
+                width="100%"
+                height="2000px"
```
Comment (sirpy) [inline src/pages/DatastudioDashboard/index.tsx:23]: why 2000?

**Review by sirpy — CHANGES_REQUESTED** [review]
please share a screenshot of how it looks

File: src/pages/DatastudioDashboard/index.tsx (hunk 2 above)
Comment (mukulshakya) [reply src/pages/DatastudioDashboard/index.tsx:23]: Perfect value is 9**px, i think this was left will update it as 2000px doesn't change anything on styling

File: src/pages/DatastudioDashboard/index.tsx (hunk 1 above)
Comment (mukulshakya) [reply src/pages/DatastudioDashboard/index.tsx:24]: Sure will refactor it a bit

**Comment by mukulshakya** [comment]
Hey @sirpy, Please have a look at the screenshots

image
image
image

**Comment by sirpy** [comment]
@mukulshakya can you explain why 9** is the perfect value?

**Review by sirpy — APPROVED** [review]
why is 924 the best value?

**Comment by mukulshakya** [comment]
> why is 924 the best value?

@sirpy This value is provided from datastudio itself

**Pull request merged**